MoltenVK/MoltenVK.xcodeproj/project.pbxproj

```diff
@@ -2561,6 +2561,7 @@
 				GENERATE_INFOPLIST_FILE = YES;
 				LD_DYLIB_INSTALL_NAME = "@rpath/${PRODUCT_NAME}.framework/${PRODUCT_NAME}";
 				OTHER_LDFLAGS = (
+					"-ld_classic",
```

Comment:
You've only applied this to the `MoltenVK-tvOS-dynamic` target. Does this issue only affect dynamic linking? Otherwise, what about the `MoltenVK-tvOS-static` target?

Also, the linked [Apple support question](https://forums.developer.apple.com/forums/thread/742470) indicates this was affecting only up to iOS 12. Does the issue being fixed here only affect older versions of tvOS? If so, how far back does the current Xcode config (without `-ld_classic`) work? Presumably, at some point in the future, once the new linker works properly for all versions of tvOS we support (currently vsOS 13+), we'd want to walk this change back.